content/chapitres/cd.adoc

```diff
@@ -36,8 +69,6 @@ Continuous Deployment
 
 * Version "avancée" de la livraison continue:
 ** Chaque changement *est* déployé en production, de manière *automatique*
-* Question importante: En avez-vous besoin ?
-** Avez-vous les mêmes besoin que Amazon Google ou Netflix ?
```

Comment:
I highly disagree with that point: CD should be instated as soon as possible in a company life. It has so many benefits even for a small company. That's why I removed it.

Reply:
We need to have a discussion on this topic, but let's keep it removed for now.